src/Microsoft.AspNetCore.Mvc.Core/Authorization/AuthorizeFilter.cs

```diff
@@ -141,28 +160,17 @@ public AuthorizeFilter(string policy)
                     if (context.Filters[i] is AuthorizeFilter authorizeFilter)
                     {
                         builder = builder ?? new AuthorizationPolicyBuilder(effectivePolicy);
```

Comment:
this isn't required anymore? as you are instantiating few lines up?

Reply:
yeah good catch, I was trying to simplify some of this, yup builder is always initialized now